engine-protocol/src/main/proto/engine_control.proto

```diff
@@ -0,0 +1,64 @@
+syntax = 'proto3';
+package engine_protocol;
+
+option java_multiple_files = false;
+option java_package = "io.camunda.zeebe.process.test.engine.protocol";
+
+message StartEngineRequest {}
+
+message StartEngineResponse {}
+
+message StopEngineRequest {}
+
+message StopEngineResponse {}
+
+message ResetEngineRequest {}
+
+message ResetEngineResponse {}
```

Comment:
❓ Did you consider taking the approach we use in Zeebe, i.e. define a common command message with a type property? If so, why this approach over the other?

Reply:
Not sure what you mean here. In Zeebe we do the same thing as I did here looking at `gateway.proto`. Care to elaborate?

Reply:
👍 Oops. I seem to have mixed-up the gateway protocol (gRPC) and the broker's protocol (SBE). 

😅 I guess it was Friday afternoon